cpp/src/cones.cpp

```diff
@@ -354,3 +354,86 @@ LinearOperator dprojection(const Vector &x, const std::vector<Cone> &cones,
   }
   return block_diag(lin_ops);
 }
+
+void _op_into_dense(MatrixRef &D_block, LinearOperator D_op) {
```

Comment:
This should probably take a const reference to the LinearOperator

Reply:
Updated